### ISSUE TITLE
::class docs says it resolves to fully qualified namespace but actually only qualified namespace

### ISSUE THREAD

**Issue by MrsKensington** [php/doc-en#2138]
From manual page: https://php.net/language.oop5.constants it says...

> The special ::class constant allows for fully qualified class name resolution at compile time, this is useful for namespaced classes:

With https://www.php.net/manual/en/language.namespaces.rules.php saying that a fully qualified namespace is...

> an identifier with a namespace separator that begins with a namespace separator, such as \Foo\Bar. The namespace \Foo is also a fully qualified name

However using `::class` on a namespaced class doesn't include the initial namespace separator in its output.

```
[root@TPS-NUCI5-DEMO1 ~]# php -a
Interactive shell

php > require_once("vendor/autoload.php");
php > var_dump(\Twig\Loader\FilesystemLoader::class);
php shell code:1:
string(28) "Twig\Loader\FilesystemLoader"
```

Either `::class` is behaving wrong or the documentation shouldn't say it resolves to a fully qualified class name.

**Comment by cmb69** [comment]
Indeed, that wording is not quite right; it probably was taken from the [respective RFC](https://wiki.php.net/rfc/class_name_scalars), which states:

> This RFC proposes to add a new ClassName::class syntax, which provides the fully qualified class name as a string.

However, the accompanying examples show that it is not really about FQNs.
